src/main/groovy/com/bugsnag/android/gradle/BugsnagVariantOutputTask.groovy

```diff
@@ -45,11 +47,18 @@ class BugsnagVariantOutputTask extends DefaultTask {
     }
 
     private File guessManifestDir(File directory, String split) {
+        String taskName = "bugsnagSplitsInfo${BugsnagPlugin.taskNameForVariant(variant)}"
+        def task = project.tasks.findByName(taskName)
+
+        if (task == null) {
+            throw new IllegalStateException("No task found matching ${taskName}")
```

Comment:
I think the same issue would apply here regarding throwing for Bugsnag tasks failing.